app/controllers/api/base_controller.rb

```diff
@@ -98,8 +98,9 @@ def process_resource_error(options = { })
     end
 
     def process_success(response = nil)
+      render_status = request.post? ? 201 : 200
```

Comment:
Last nitpick: is it possible to use symbols here, like elsewhere?  e.g. :ok, :created (guessing)